### PR TITLE
FIX: [Web App] Convert the style guide from figma design into base styles

### LINKED ISSUE #40: Convert the style guide from figma design into base styles
#### Description and expected solution
- [ ] Convert the style guide from our figma design into base styles:
- Colors
- Typography
- Fonts

> Note 1: For now just use and work closely with the naming convention from @Geefundesign's designs on figma

> Note 2: we have a base style file called var.css, please make use of css variables as needed

> Note 3: Ensure to conduct a basic test that the styles added are working

> Note 4: there's already a structure for our css styles. Please do not alter the structure, just flow with the structure when adding styles or files. If you have any concerns or suggestions related to stucture let's discuss that first before making any change that totally changes the css scaffold structure.

### PR DESCRIPTION
**Pull request by samuko-things**
**This pull request makes the following changes**
* Fixes code-collabo/collabo-community-app#40 

**General checklist**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me

**Testing checklist**

- [x] Convert the style guide from our figma design into base styles:
- Colors
- Typography
- Fonts

Ping @code-collabo/web-app

### CODE REVIEW

**Comment by samuko-things** [comment]
@Ifycode @Festusali @Geefundesign

I have added the base color and typography style guide from the community app Figma design as variables in the var.css file as instructed. I currently remain the underline part. I have also tested them externally.

please do check it out and let me know what you think.

**Comment by Ifycode** [comment]
Just tag @code-collabo/community-mentors next time.
Reviewing your PR...

File: client/src/styles/base/var.css [hunk 1]
```diff
@@ -1,5 +1,101 @@
...
+
+
+
+  /*===========TYPOGRAPHY==============*/
+  /*   used with the font property    */
+  /* https://www.w3schools.com/cssref/pr_font_font.php */
+
+  --typo-semi-bold-h1: 600 48px/65.37px 'Open Sans';
+  --typo-semi-bold-h2: 600 34px/46.30px 'Open Sans';
+  --typo-semi-bold-h3: 600 24px/32.68px 'Open Sans';
```
Comment (Ifycode) [inline client/src/styles/base/var.css]: **_Review comment 1:_**
Nice work so far @samuko-things 
Help change `--typo` to `--type` everywhere it is used.

File: client/src/styles/base/var.css [hunk 2]
```diff
@@ -1,5 +1,101 @@
...
+  --color-neutral-400: #121518; /*main*/
+
+  /*==================================*/
+
+
+
+
+  /*===========TYPOGRAPHY==============*/
+  /*   used with the font property    */
+  /* https://www.w3schools.com/cssref/pr_font_font.php */
```
Comment (Ifycode) [inline client/src/styles/base/var.css:54]: **_Review comment 2:_**
Interesting. This is my first time seeing font size and line height written in this manner. The link is helpful @samuko-things 

What about if the owners of that reference site changes their link or it become none existent?

You can just help add a comment below the reference link that gives an idea of the format ou used e.g
````
/* --type-style-format: font-weight font-size/line-height font-family */
````

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: client/src/styles/base/var.css (hunk 1 above)
Comment (samuko-things) [reply client/src/styles/base/var.css]: ok

File: client/src/styles/base/var.css (hunk 2 above)
Comment (samuko-things) [reply client/src/styles/base/var.css:54]: ok will do

**Comment by samuko-things** [comment]
> Just tag @code-collabo/community-mentors next time. Reviewing your PR...

ok noted.

**Comment by samuko-things** [comment]
I have just added a style guide for the underline functionality based on the figma design to be used with text-decoration property. I've also tested it. please do check it out and let me know what you think.

**Review by Ifycode — APPROVED** [review]
Awesome work @samuko-things
Approved.

**Pull request merged**